src/modules/users/user-list/users.js

```diff
@@ -0,0 +1,89 @@
+(function() {
+	"use strict";
+
+	angular
+		.module("TendrlModule")
+		.component("users",{
+
+			restrict: "E",
+			templateUrl: "/modules/users/user-list/users.html",
+			bindings: {},
+			controller: userController,
+			controllerAs: "userCntrl"
+		});
+
+	/*@ngInject*/
+	function userController($rootscope, $scope, $interval, $state, $timeout, $filter, orderByFilter, config, userStore) {
+
+		var vm = this,
+			userTimer;
+
+		//vm.userStatus = [];
+		vm.isDataLoading = true;
+
+		init();
+
+		function init() {
+			userStore.getUserList()
+				.then(function(data){
+					vm.userList = data;
+					vm.isDataLoading = false;
+					startTimer();
+				});
```

Comment:
@brainfunked @nthomas-redhat @anivargi Do we need to do polling for user list API response.